utils.js

```diff
@@ -3,6 +3,7 @@
 const {
   ArrayPrototypeFind,
   ObjectEntries,
+  ObjectPrototypeHasOwnProperty: ObjectHasOwn,
```

Comment:
Minor. Is this (still) needed in node? I accidentally imported `ObjectHasOwn` and it worked fine here in parseArgs, but we don't have real primordials!

Reply:
it's needed in case we want to backport it to an older node that lacks `Object.hasOwn`.

Reply:
My slight concern is it means we are masking the new routine and using the old routine.

Reply:
That's intentional; by spec they're almost exactly identical (except for which exception gets thrown first when the args are invalid), and once "node without Object.hasOwn" goes EOL, nothing will use ObjectPrototypeHasOwnProperty anymore, and this renaming minimizes that future code churn.

Reply:
Thanks for explanation.